LongWarOfTheChosen/Src/LW_Overhaul/Classes/LWTacticalMissionUnitSpawner.uc

```diff
@@ -499,7 +504,11 @@ static function LoadLiaisonFromOutpost(XComGameState_LWOutpost Outpost,
 		{
 			ItemState = XComGameState_Item(NewGameState.CreateStateObject(class'XComGameState_Item', ItemReference.ObjectID));
 			NewGameState.AddStateObject(ItemState);
+			//Issue #159, this is needed now for loading units from avenger to properly update gamestate.
+			ItemState.BeginTacticalPlay(NewGameState);
 		}
+		//NumAbilities = Unit.GatherUnitAbilitiesForInit(NewGameState, PlayerState).Length;
+		//`Log("Number of unit abilities: " $ NumAbilities);
```

Comment:
Perhaps remove this code and the local variable you created. Or enable it for logging purposes.

Reply:
ahh I forgot to remove that. I'll do that

Reply:
done